go.mod

```diff
@@ -38,4 +38,6 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // avoids case-insensitive import collision: "github.com/googleapis/gnostic/openapiv2" and "github.com/googleapis/gnostic/OpenAPIv2"
 )
 
+replace github.com/codeready-toolchain/api => github.com/rajivnathan/api v0.0.0-20201218021903-cf2c667c66b9
```

Comment:
Friendly reminder to remove that :)

Reply:
Done in https://github.com/codeready-toolchain/member-operator/pull/231/commits/dc65c9b62207990d5524893de4c95d952972765c